### PR TITLE
Solution for problem 27 in JS

### PR DESCRIPTION
**Pull request by alnavarroh**
approve pls 👀

### CODE REVIEW

**Review by Eduardo-93 — APPROVED** [review]
lgtm

**Review by Bydel93 — COMMENTED** [review]
lgtm 👍🏼

File: algorithm_exercises/27. Remove Element/solution.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {number[]} nums
+ * @param {number} val
+ * @return {number}
+ */
+var removeElement = function (nums, val) {
+    let length = nums.length
```
Comment (Darova93) [inline algorithm_exercises/27. Remove Element/solution.js]: Since length is not modified it can be a constant. 
Same for the function

**Review by Darova93 — CHANGES_REQUESTED** [review]

File: algorithm_exercises/27. Remove Element/solution.js (hunk 1 above)
Comment (alnavarroh) [reply algorithm_exercises/27. Remove Element/solution.js]: addresed in latest commit

**Review by danielalbinoms — APPROVED** [review]
nice

**Review by Darova93 — APPROVED** [review]

**Review by danielalbinoms — APPROVED** [review]
fancy

**Pull request merged**